.travis.yml

```diff
@@ -3,7 +3,7 @@ os:
 
 language: go
 go:
-  - 1.8
+  - 1.x
```

Comment:
holochain requires at least version 1.8, which is why I set this that way.  Is there a way to specify some version or above?

Reply:
@zippy i added 1.8.x and 1.9.x instead 1.x